### PR TITLE
Add  params class for ExtracText class

### PR DESCRIPTION
**Pull request by JorgeRCV**

### CODE REVIEW

File: src/main/java/org/fundacion/jala/converter/controller/ExtractTextController.java [hunk 1]
```diff
@@ -10,15 +10,18 @@
...
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
 import org.springframework.web.multipart.MultipartFile;
 import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+
 import java.io.IOException;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
-import org.fundacion.jala.converter.service.ExtractText;
```
Comment (GustavoZHA) [inline src/main/java/org/fundacion/jala/converter/controller/ExtractTextController.java:25]: Delete Blank line

File: src/main/java/org/fundacion/jala/converter/controller/ExtractTextController.java [hunk 2]
```diff
@@ -38,7 +41,7 @@ public String uploadFile(final @RequestParam("file")MultipartFile file,
         LOGGER.info("start");
         String filename = file.getOriginalFilename();
         String storagePath = fileStorageService.uploadFile(file);
-        ExtractText extractText = new ExtractText(language, storagePath, nameOutput);
+        ExtractText extractText = new ExtractText(new ExtractTextParameter( storagePath,language, nameOutput));
```
Comment (GustavoZHA) [inline src/main/java/org/fundacion/jala/converter/controller/ExtractTextController.java]: add space in "storagePath,language,"

File: src/main/java/org/fundacion/jala/converter/models/parameter/ExtractTextParameter.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    private String resultFile;
+    public ExtractTextParameter(String newFilePath) {
+        super(newFilePath);
+    }
+    public ExtractTextParameter(final String newFilePath, final String language,final String resultFile){
+        super(newFilePath);
+        this.language = language;
+        this.resultFile = resultFile;
+    }
+    public ExtractTextParameter() {
```
Comment (Paola-Xaq) [inline src/main/java/org/fundacion/jala/converter/models/parameter/ExtractTextParameter.java:30]: JavaDoc is missing

File: src/main/java/org/fundacion/jala/converter/models/parameter/ExtractTextParameter.java [hunk 4]
```diff
@@ -0,0 +1,35 @@
+package org.fundacion.jala.converter.models.parameter;
```
Comment (GustavoZHA) [inline src/main/java/org/fundacion/jala/converter/models/parameter/ExtractTextParameter.java:11]: Add Copyright

File: src/main/java/org/fundacion/jala/converter/models/parameter/ExtractTextParameter.java [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+        super(newFilePath);
+    }
+    public ExtractTextParameter(final String newFilePath, final String language,final String resultFile){
+        super(newFilePath);
+        this.language = language;
+        this.resultFile = resultFile;
+    }
+    public ExtractTextParameter() {
+    }
+
```
Comment (GustavoZHA) [inline src/main/java/org/fundacion/jala/converter/models/parameter/ExtractTextParameter.java]: Apply checkStyle  all class.

**Review by Paola-Xaq — COMMENTED** [review]
Fix Javadocs missing
add Copyrigthand author

File: src/main/java/org/fundacion/jala/converter/models/parameter/ExtractTextParameter.java [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+    public ExtractTextParameter(final String newFilePath, final String newLanguage, final String newResultFile) {
+        super(newFilePath);
+        this.language = newLanguage;
+        this.resultFile = newResultFile;
+    }
+
+    public ExtractTextParameter() {
+    }
+    /**
```
Comment (Paola-Xaq) [inline src/main/java/org/fundacion/jala/converter/models/parameter/ExtractTextParameter.java:32]: Specify the Javadoc,  add in third person what does the method do, what type of data is return and what parameter type is needed.

Hope you understand mi comment :).

**Review by Paola-Xaq — APPROVED** [review]
Just need to add more detail to Javadoc, everything seems fine after that :).

**Review by JessickaMoya — APPROVED** [review]

**Pull request merged**